pkg/ocm/accounts_mgmt.go

```diff
@@ -81,30 +83,44 @@ func (a accountsMgmt) GetSubscription(ctx context.Context, subscriptionID strfmt
 	return responseVal, nil
 }
 
+func (a accountsMgmt) updateSubscription(ctx context.Context, subscriptionID strfmt.UUID, sub *amgmtv1.Subscription, err error) error {
+	if err != nil {
+		a.client.logger.Error(ctx, "Failed to create subscription request. Error: %v", err)
+		return err
+	}
+	return a.sendSubscriptionUpdateRequest(ctx, subscriptionID, sub)
+}
+
 // This function updates all the subscription's data that we didn't know at the subscription creation time
 // and we know only after installation is done such openshift_cluster_id.
 func (a accountsMgmt) UpdateSubscriptionPostInstallation(ctx context.Context, subscriptionID, openshiftClusterID strfmt.UUID) error {
 	defer commonutils.MeasureOperation("OCM-UpdateSubscriptionPostInstallation", a.client.log, a.client.metricsApi)()
 
 	sub, err := amgmtv1.NewSubscription().ExternalClusterID(openshiftClusterID.String()).Status(SubscriptionStatusActive).Build()
-	if err != nil {
-		a.client.logger.Error(ctx, "Failed to create subscription request. Error: %v", err)
-		return err
-	}
-
-	return a.sendSubscriptionUpdateRequest(ctx, subscriptionID, sub)
+	return a.updateSubscription(ctx, subscriptionID, sub, err)
 }
 
 func (a accountsMgmt) UpdateSubscriptionDisplayName(ctx context.Context, subscriptionID strfmt.UUID, displayName string) error {
 	defer commonutils.MeasureOperation("OCM-UpdateSubscriptionDisplayName", a.client.log, a.client.metricsApi)()
 
 	sub, err := amgmtv1.NewSubscription().DisplayName(displayName).Build()
-	if err != nil {
-		a.client.logger.Error(ctx, "Failed to create subscription request. Error: %v", err)
-		return err
+	return a.updateSubscription(ctx, subscriptionID, sub, err)
```

Comment:
I think that we need to add a log here - the same as we have in UpdateSubscriptionConsoleUrlIfNeeded

Reply:
Done.